webapp/channels/src/components/integrations/outgoing_oauth_connections/installed_outgoing_oauth_connections.tsx

```diff
@@ -130,8 +129,8 @@ const InstalledOutgoingOAuthConnections = (props: Props) => {
                 />
             }
             emptyTextSearch={
-                <FormattedMarkdownMessage
-                    id='installed_outgoing_oauth_connections.emptySearch'
+                <FormattedMessage
+                    id='installedOutgoingOauthConnections.blankSearch'
```

Comment:
since the defaultMessage wasnt changed, we can keep the same id

Reply:
Let me push it!